### DISCUSSION TITLE
If developing and deploying on Ubuntu, do I need to specify amd64?

### DISCUSSION THREAD

**Discussion by viktorsmari** [basecamp/kamal#520] [General]
From the documentation, it says:

"If you’re developing on the same architecture as the one you’re deploying on, you can speed up the build by forgoing both multi-arch and remote building"
```
builder:
  multiarch: false
```

My Ubuntu is running on `amd64` according to `dpkg --print-architecture`

Do I **also** need to add the `local: arch: amd64`? in the `config/deploy.yml`:
```
builder:
  multiarch: false
  local:
    arch: amd64
```

Does Kamal detect which architecture I am using, or does it always **assume** everyone is using `arm64`?

**Comment by frenkel** [comment]
I believe multiarch is enabled by default, so it assumes everyone wants to support both `amd64` and `arm64`. It's rather easy to test though, just run `kamal build  push` and see if only amd64 or both are listed in the output.
